### PR TITLE
fix text in portuguese

### PR DESCRIPTION
**Pull request by gloriaabreualves**
fix text in portuguese.

- [ x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [ x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x ] My pull request targets the `master` branch of freeCodeCamp.
- [x ] None of my changes are plagiarized from another source without proper attribution.
- [x ] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x ] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

**Review by gabryellr — APPROVED** [review]

**Comment by RandellDawson** [comment]
@CleoAguiar Could you review this PR's changes match the [current English version of the documentation](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md)?

Thank you

**Comment by CleoAguiar** [comment]
> @CleoAguiar Could you review this PR's changes match the [current English version of the documentation](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md)?
> 
> Thank you

This PR's changes do match the the old version of the documentation

**Comment by RandellDawson** [comment]
> This PR's changes do match the the old version of the documentation

@CleoAguiar What would need to be changed in this PR to make it match the current English version of the document?  If you tell me which lines need to be changed, I will add a new commit with the changes and merge the PR.

File: docs/i18n-languages/portuguese/CONTRIBUTING.md [hunk 1]
```diff
@@ -15,13 +15,13 @@ Divirta-se contribuindo 🎉!
 
 ## Esses são alguns jeitos divertidos de contribuir
 
-Você pode contribuir para qualquer área do seu interesse:
+Você pode contribuir em qualquer área do seu interesse:
 
 1. [Contribua para essa base de códigos abertos.](#contribua-para-essa-base-de-códigos-abertos)
```
Comment (CleoAguiar) [inline docs/i18n-languages/portuguese/CONTRIBUTING.md:20]: Need to fixed list itens

File: docs/i18n-languages/portuguese/CONTRIBUTING.md [hunk 2]
```diff
@@ -15,13 +15,13 @@ Divirta-se contribuindo 🎉!
 
 ## Esses são alguns jeitos divertidos de contribuir
```
Comment (CleoAguiar) [inline docs/i18n-languages/portuguese/CONTRIBUTING.md:16]: Need to fixed list itens.

File: docs/i18n-languages/portuguese/CONTRIBUTING.md [hunk 3]
```diff
@@ -57,7 +57,7 @@ Nós agradecemos sua ajuda na escrita desses artigos. Você não precisa ser um
 
 Para ajudar, encontre um `artigo provisório` na [página de Guias](https://www.freecodecamp.org/guide), escreva o artigo, e então abra um <i>pull request</i> (PR) para substituir o provisório pelo seu artigo. Um [<i>pull request</i>](https://help.github.com/articles/about-pull-requests/) é a forma como você vai sugerir mudanças. Ele permite que outros saibam das suas mudanças, revisem-nas e as adotem.
 
-Se você não consegue encontrar um artigo provisório do tópico sobre o qual você gostaria de escrever, você pode abrir um PR que cria o artigo provisório e então incluir o rascunho do seu artigo.
```
Comment (CleoAguiar) [inline docs/i18n-languages/portuguese/CONTRIBUTING.md:60]: Need to updated the markdonw headings in above session "Contribua para essa base de códigos abertos"

**Pull request closed without merging**